content/kubermatic/v2.14/data/kubermaticConfiguration.yaml

```diff
@@ -8,11 +8,12 @@ spec:
   api:
     # AccessibleAddons is a list of addons that should be enabled in the API.
     accessibleAddons:
-    - node-exporter
+      - node-exporter
+      - gatekeeper
```

Comment:
This needs to be applied in the master version too

Reply:
Related to the master kubermaticConfiguration.yaml file. Applies to all changes below.